src/Invictus.Testing/LogicAppTriggerNotFoundException.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace Invictus.Testing
+{
+    /// <summary>
+    /// Exception thrown when no trigger can be found for a given logic app.
+    /// </summary>
+    [Serializable]
+    public class LogicAppTriggerNotFoundException : Exception
+    {
```

Comment:
This exception is lacking useful information if you ask me; can we add the following please?
- `SubscriptionId`
- `ResourceGroup`
- `LogicAppName`

Reply:
Done